seacrowd/sea_datasets/vivos/vivos.py

```diff
@@ -0,0 +1,204 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{luong-vu-2016-non,
+    title = "A non-expert {K}aldi recipe for {V}ietnamese Speech Recognition System",
+    author = "Luong, Hieu-Thi  and
+      Vu, Hai-Quan",
+    editor = "Murakami, Yohei  and
+      Lin, Donghui  and
+      Ide, Nancy  and
+      Pustejovsky, James",
+    booktitle = "Proceedings of the Third International Workshop on Worldwide Language Service
+    Infrastructure and Second Workshop on Open Infrastructures and Analysis Frameworks for
+    Human Language Technologies ({WLSI}/{OIAF}4{HLT}2016)",
+    month = dec,
+    year = "2016",
+    address = "Osaka, Japan",
+    publisher = "The COLING 2016 Organizing Committee",
+    url = "https://aclanthology.org/W16-5207",
+    pages = "51--55",
+    abstract = "In this paper we describe a non-expert setup for Vietnamese speech recognition
+    system using Kaldi toolkit. We collected a speech corpus over fifteen hours from about fifty
+    Vietnamese native speakers and using it to test the feasibility of our setup. The essential
+    linguistic components for the Automatic Speech Recognition (ASR) system was prepared basing
+    on the written form of the language instead of expertise knowledge on linguistic and phonology
+    as commonly seen in rich resource languages like English. The modeling of tones by integrating
+    them into the phoneme and using the phonetic decision tree is also discussed. Experimental
+    results showed this setup for ASR systems does yield competitive results while still have
+    potentials for further improvements.",
+}
+"""
+
+_DATASETNAME = "vivos"
+
+_DESCRIPTION = """\
+VIVOS is a Vietnamese speech corpus consisting of 15 hours of recording speech prepared for
+Automatic Speech Recognition task. This speech corpus is collected by recording speech data
+from more than 50 native Vietnamese volunteers.
+"""
+
+_HOMEPAGE = "https://zenodo.org/records/7068130"
+
+_LANGUAGES = {"vie": "vi"}
```

Comment:
Could you please make this part a `list` instead of a `dict`?

Reply:
Pushed a commit to change the `_LANGUAGES` dict into a list. Should be PR ready.